### PR TITLE
Pull request for feature branch

### PR DESCRIPTION
**Pull request by uno36**
This is a mini project for HTM & CSS
I added index.html, styles.css and test.md
Ignored test.md
Created a MIT License from template and added linter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # 📖 Hello-Microverse-Project <a name="about-project"></a>
 
-> The project helps to learn HTML, CSS and the addition of necessary linters 
+> The project helps to learn HTML, CSS and the addition of necessary linters. 
 
 **Hello-Microverse** is a Project
```
Comment (AbayomiOlaoye) [inline README.md:7]: - You are almost done with this project, @uno36 :wink: 
- - [ ] :+1: Like I said, you have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have contents that make your project look professional to visitors or users. Kindly check [this README template](https://github.com/microverseinc/readme-template) again and see all the important sub-titles that you are missing and their respective details.
  - [ ] * The only **optional** section is FAQ.
  * You can delete the live demo section since you are not deploying your application now.
  * Kindly add the **Table of Contents** to your ReadMe.
  * Also, kindly include the **Key features** of the project.
* It will be more customized to **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
* In Summary, the only section you can leave out is the `FAQs` because it is optional.

Keep doing an amazing job, Micronaut! :clap:

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @uno36  :wave: , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

:alarm_clock: Please do not create another **Pull Request** after implementing the required changes. Once you have pushed your changes, just submit this very pull request link again - not a new one. This is very important so that you can get consistent feedback on your code :pray: :pleading_face: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by uno36** [comment]
I have updated Readme file
I added table of contents
I also added key features

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @uno36, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
